src/Vizkit3DWidget.hpp

```diff
@@ -158,6 +159,7 @@ namespace vizkit3d
             Q_ENUMS( CAMERA_MANIPULATORS )
             Q_PROPERTY( QStringList manipulator READ getAvailableCameraManipulators WRITE setCameraManipulator )
             Q_PROPERTY( double transformerTextSize READ getTransformerTextSize WRITE setTransformerTextSize );
+            Q_PROPERTY( bool backCulling READ isBackCulling WRITE setBackCulling);
```

Comment:
@planthaber  the backCulling property in Vizkit3DConfig is already here :)

Reply:
ok, overlooked that one